packages/core/src/components/textarea/textarea-vars.scss

```diff
@@ -12,6 +12,7 @@
   --tds-textarea-disabled-background: var(--tds-textarea-disabled-background-primary);
   --tds-textarea-disabled-placeholder: var(--tds-grey-400);
   --tds-textarea-disabled-label: var(--tds-grey-400);
+  --tds-textarea-disabled-label-inside-color: var(--tds-grey-700);
```

Comment:
Regarding the discussion had earlier today, should this be changed to grey-800?

Reply:
Possibly but we did not confirm that with Charlotte, only the outside one. 
Can you please double-check with her? 
Suck that Figma is not up-to-date and a source of truth.

Reply:
It has now been verified with Charlotte that label inside should also be grey-800 when the component is disabled :)